src/metabase/query_processor/util/add_alias_info.clj

```diff
@@ -289,9 +289,9 @@
     ;; suffix to escaped identifiers.)
     ;;
     ;; We'll have to look into this more in the future. For now, it seems to work for everything we try it with.
-    (and join-alias (not join-is-this-level?)) (prefix-field-alias join-alias field-name)
     (and join-is-this-level? alias-from-join)  alias-from-join
     alias-from-source-query                    alias-from-source-query
```

Comment:
I'm not quite sure if this is the right order, if the reverse order would be better or if it doesn't matter. (I guess it doesn't matter.)